### PR TITLE
[MAINT] - Edits to Dashboard tutorial

### PR DESCRIPTION
**Pull request by trallard**
This PR adds the following:

- :memo: Update dashboard tutorial
- :truck: Reorg doc images - to keep them tidy I have created folders that mimic the current docs structure

### CODE REVIEW

File: docs/docs/tutorials/creating-cds-dashboard.md [hunk 1]
```diff
@@ -3,45 +3,44 @@ id: creating-cds-dashboard
 title: Create and Deploy a dashboard
```
Comment (kcpevey) [inline docs/docs/tutorials/creating-cds-dashboard.md:3]:
```suggestion
title: Create and deploy a dashboard
```

File: docs/docs/tutorials/creating-cds-dashboard.md [hunk 2]
```diff
@@ -3,45 +3,44 @@ id: creating-cds-dashboard
...
 ---
 
-# Create a dashboard with `Panel` and deploy it with `CDS Dashboards`
+# Create and deploy a dashboard with `Panel` and `CDS Dashboards`
```
Comment (kcpevey) [inline docs/docs/tutorials/creating-cds-dashboard.md:6]: This main header is replaced by the docusaurus `title` above on the page tiles. 
image

I'm happy with this title, but I was thinking you wanted to make it more descriptive?

Comment (trallard) [reply docs/docs/tutorials/creating-cds-dashboard.md:6]: I still needed to come back and add a description

File: docs/docs/tutorials/creating-cds-dashboard.md [hunk 3]
```diff
@@ -3,45 +3,44 @@ id: creating-cds-dashboard
...
 
-## What is a dashboard?
+## Introduction
```
[the remark this answers is not in the document]
Comment (trallard) [reply docs/docs/tutorials/creating-cds-dashboard.md:8]: Not needed with the description

**Comment by trallard** [comment]
This should be ready @kcpevey

**Review by kcpevey — APPROVED** [review]
Looks great!

**Pull request merged**